devdocs/process.md

```diff
@@ -129,7 +129,9 @@ This workflow is an adaptation of the [GitHub flow](https://guides.github.com/in
      the relevant issue name and include the issue number as well.<br>
      e.g. ` Incorrect error message when adding an existing instructor #1760`<br>
      In the PR description, mention the issue number in this format: `Fixes #1760`. 
-     Doing so will create an automatic reference from the issue to the pull request.<br>
+     Doing so will achieve an automatic reference (by simply mentioning the issue number with the #).<br>
+     To close an issue from a PR, all you have to do is include the special keyword syntax (eg. "fixes #5") 
+     in the body of your Pull Request. And the referenced issue will automatically be closed when the PR is merged into the default branch.<br>
```

Comment:
We can keep this description brief (as @wkurniawan07 has suggested) but it is ok to hyper link to the relevant section of GitHub's official documentation.